config

```diff
@@ -500,12 +502,7 @@ case "$GUESSOS" in
 	if [ "$ISA64" = "1" -a -z "$KERNEL_BITS" ]; then
 	    echo "WARNING! If you wish to build 64-bit library, then you have to"
 	    echo "         invoke 'KERNEL_BITS=64 $THERE/config $options'."
-	    if [ "$DRYRUN" = "false" -a -t 1 ]; then
-	      echo "         You have about 5 seconds to press Ctrl-C to abort."
-	      # The stty technique used elsewhere doesn't work on
-	      # MacOS. At least, right now on this Mac.
-	      sleep 5
-	    fi
+            sleep $SLEEPTIME
```

Comment:
Jagged formatting.

Anyway, the original report is about removal *any* delay specifically in MacOS X case. Basically the original "interface", the one referred to as "stty technique used elsewhere", is if you don't want to wait, just press return. But apparently it didn't work on MacOS X, so somebody replaced it with sleep 5. That of course is annoying. And so would any one that you can't do anything about. But note that #2167 actually offers modified "stty technique" that works...

Reply:
BTW, this case is executed only on older MacOS X versions, Newer ones fall into next `x86_64-apple-darwin*)` category.

Reply:
The formatting was space-vs-tab; fixed.
Your portable technique is clever.  More than I want to do.  Should maybe we just close both
of these?